packages/styles/src/sass/components/link/_index.scss

```diff
@@ -0,0 +1,80 @@
+.cbp-link {
+  font-weight: var(--cbp-font-weight-regular);
```

Comment:
This should not have to be set explicitly - it should inherit from its context.

Reply:
fixed.